src/View/Form/EntityContext.php

```diff
@@ -279,6 +265,12 @@ public function val(string $field, array $options = [])
 
         if ($entity instanceof EntityInterface) {
             $part = end($parts);
+
+            $val = $entity->getInvalidField($part);
```

Comment:
This method isn't part of the `EntityInterface` you'd need to do a check for `InvalidPropertyInterface` here as well.